.github/workflows/ci.yaml

```diff
@@ -45,6 +45,19 @@ jobs:
 
       - name: Install Microsoft ODBC Drivers
         run: sudo ACCEPT_EULA=Y apt-get install msodbcsql18 -y
+      - name: Free additional space
```

Comment:
Just curious, why is this needed?

Reply:
We've had quite a few failures because we go beyond the 20gb or so that Github allocates

Reply:
Oh :eyes: